utils/onSubmit.ts

```diff
@@ -197,7 +200,6 @@ export function onSubmit({
               success: true,
             },
           };
-          console.log(result);
```

Comment:
Why remove logging?

Reply:
I will revert.

Reply:
I was looking for my logs that I had lingering and removed them. I take it as blindly doing it :D

Reply:
https://github.com/Snowfork/snowbridge-app/pull/23/commits/a385248bc4014588d7c12fee7a4817fb2f3cccf8